networkx/algorithms/bridges.py

```diff
@@ -202,15 +217,58 @@ def all_bridges(G):
         startnode = allnodes[0]
         # run the DFS
         dfs(startnode, -1)
+    # sort the bridges ascending order (for consistency in output)
+    for edge in bridges:
+        if edge[0] > edge[1]:
+            temp = edge[1]
+            edge[1] = edge[0]
+            edge[0] = temp
+    bridges = sorted(bridges)
     return bridges
 
 
+def local_bridges_exist(G):
+    """ Checks if any local bridges exist in this network. We will simply call
+    the all_local_bridges() function with the first-match stop parameter.
+    Since the search for local bridges will terminate after finding one
+    instance, this function is faster if we just want to know if at least
+    one local bridge exists in the network and what it is.
+
+    Parameters
+    ----------
+    G : Undirected Graph object
+
+    Returns
+    ----------
+    boolean
+        True if we found at least one bridge.
+        False otherwise.
+
+    Examples
+    --------
+    >>> G = nx.cycle_graph(5)
+    >>> local_bridges_exist(G)
+    True
+    >>> G = nx.complete_graph(5)
+    >>> local_bridges_exist(G)
+    False
+
+    Notes
+    ----------
+    This function can be useful to quickly determine whether or not local
+    bridges exist in a given network.
+    """
+
+    results = all_local_bridges(G, first_match=True)
+    if len(results["spans"]) > 0:
+        return True
+    else:
+        return False
+
+
 def bridges_exist(G):
     """ Checks if any bridges exist in this network. We will simply call the
-    all_bridges() function with the first-match stop parameter. Since the
```

Comment:
Don't forget that you can use Markdown syntax in your docstrings, which makes the generated docs look really nice. As an example: \`all_bridges()\` (with back-ticks) becomes `all_bridges()`.